Makefile

```diff
@@ -33,6 +35,8 @@ IMAGE_NAME ?= ramen
 IMAGE_TAG ?= latest
 IMAGE_TAG_BASE = $(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
 RBAC_PROXY_IMG ?= "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
+OPERATOR_SUGGESTED_NAMESPACE ?= ramen-system
```

Comment:
Attempting to understand if non-upstream builds can specify additional ENV vars as in here, such that this scheme can be used.